test/tests_for_db/ee2_MongoUtil_test.py

```diff
@@ -57,6 +58,39 @@ def test_init_ok(self):
         mongo_util = self.getMongoUtil()
         self.assertTrue(set(class_attri) <= set(mongo_util.__dict__.keys()))
 
+    def test_update_jobs_success(self):
+        job = get_example_job(status=Status.created.value)
+        job2 = get_example_job(status=Status.created.value)
+        job_id1 = job.save().id
+        job_id2 = job2.save().id
+        job_ids = [job_id1, job_id2]
+        scheduler_ids = [job.scheduler_id, job2.scheduler_id]
+        jobs_to_update = list(map(JobIdPair, job_ids, scheduler_ids))
+        print(jobs_to_update)
+        assert Status.created.value in [job.status, job2.status]
+
+        # TODO RETRY FOR RACE CONDITION OF RUN/CANCEL
+
+    def test_update_jobs_failure(self):
+        job = get_example_job(status=Status.created.value)
+        job2 = get_example_job(status=Status.error.value)
+        job3 = get_example_job(status=Status.terminated.value)
+
+        job_id1 = job.save().id
+        job_id2 = job2.save().id
+        job_id3 = job3.save().id
+        job_ids = [job_id1, job_id2, job_id3]
+        scheduler_ids = [job.scheduler_id, job2.scheduler_id, job3.scheduler_id]
+        jobs_to_update = list(map(JobIdPair, job_ids, scheduler_ids))
+        print(jobs_to_update)
+        assert Status.created.value in [job.status, job2.status, job3.status]
```

Comment:
Same as above re the test not quite working

Reply:
fixed

Reply:
:+1: